### PR TITLE
completed react-effects exercise.

### PR DESCRIPTION
**Pull request by joseph-w-valdez**
https://user-images.githubusercontent.com/117682160/218186615-6d1f8799-f092-4041-81f3-ac4bc1061296.mp4

### CODE REVIEW

File: react-effects/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-unused-vars -- Remove me */
```
Comment (RobertGardner) [inline react-effects/src/index.jsx:1]: As the comment says, remove this line. :-)

File: react-effects/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  // Your code here:
+  //  - Read the items using `readItems` and update state so the list displays
+
+  useEffect(() => {
+    const app = readItems();
+    app
+      .then((value) => {
+        setItems(value);
+        setIsLoading((prev) => !prev);
```
Comment (RobertGardner) [inline react-effects/src/index.jsx:18]: I don't like this. What if `isLoading` is false when this code executes? Do you really want to toggle it to true? That doesn't make sense.

File: react-effects/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      })
+      .catch((error) => {
+        console.error(error.message);
+      });
+  }, items);
```
Comment (RobertGardner) [inline react-effects/src/index.jsx:23]: This needs to be an array of dependencies. The code in here does not depend on `items`.

**Review by RobertGardner — CHANGES_REQUESTED** [review]

File: react-effects/src/index.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState([]);
+
+  // Your code here:
+  //  - Read the items using `readItems` and update state so the list displays
+
+  useEffect(() => {
+    const app = readItems();
```
Comment (RobertGardner) [inline react-effects/src/index.jsx:13]: I suggest you chain directly from `readItems`. It makes the code easier to follow.

**Review by RobertGardner — APPROVED** [review]
This looks correct to me!

File: react-effects/src/index.jsx (hunk 4 above)
Comment (joseph-w-valdez) [reply react-effects/src/index.jsx:13]: will do!

**Pull request merged**